plugin/src/main/scala-2.12/ch/epfl/scala/profilers/tools/ScalaSettingsOps.scala

```diff
@@ -0,0 +1,6 @@
+package ch.epfl.scala.profilers.tools
+
+object ScalaSettingsOps {
+  def isScala212: Boolean = true
+  def isScala213: Boolean = false
```

Comment:
That is a bit controversial, perhaps there is a more elegant way to do these checks.

Reply:
Just for the context, https://github.com/scala/scala/pull/9575.